mobile/native/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 mod api;
 pub mod api_calculations;
+pub mod api_lndlc;
 pub mod api_model;
```

Comment:
:wrench: This module structure and naming seems a bit weird. Why is there a private `api` module and public `api_*` modules. Why not sub-modules of `api` leading to a more natural naming and structure e.g. `api::calculations`.

Also, as a nitpick, I would use `ln_dlc`  or `ln_dlc_node` as opposed to `lndlc` to be consistent with the name of the crate.